proposals/0280-Adding-new-parameter-of-requiresAudioSupport-and-BluetoothDeviceAddress.md

```diff
@@ -168,52 +174,7 @@ Due to the complexity of the flow, the developer must do the implementation care
 This proposal requires a major version change.
 Since new parameters are added, Core, iOS, Java Suite, RPC, Protocol, and HMI are affected.
 
-
 ## Alternatives considered
 
-The flow in the Proposed solution section is complicated, so the author has also considered the simplified solution below.
```

Comment:
You can update this section to reflect the reason why we decided to move forward with the flow described in the `Proposed solution` section:
```suggestion
While this alternative was considered, the Steering Committee determined this flow does not account for previous versions of the protocol, and therefore is not viable as the primary solution.
```

Reply:
Hi, @theresalech -san.
Thank you for your review.
I will modify to this sentence after adding back lines 174-219 and make a commit it.
Could you please confirm it again.
Thank you.

Reply:
@Akihiro-Miyazaki, this does not yet appear to be resolved.

Reply:
Hi, @theresalech -san.
I am very sorry for delay my response.
I corressponded it and made a commit it.
Could you please confirm it again.
Thank you.